list.go

```diff
@@ -0,0 +1,85 @@
+// +build linux
+
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"text/tabwriter"
+
+	"github.com/Sirupsen/logrus"
+	"github.com/codegangsta/cli"
+)
+
+var listCommand = cli.Command{
+	Name:  "list",
+	Usage: "lists containers started by runc with the given root",
+	Action: func(context *cli.Context) {
+
+		// preload the container factory
+		if factory == nil {
+			err := factoryPreload(context)
+			if err != nil {
+				logrus.Fatal(err)
+				return
+			}
+		}
+
+		// get the list of containers
+		root := context.GlobalString("root")
+		absRoot, err := filepath.Abs(root)
+		if err != nil {
+			logrus.Fatal(err)
+			return
+		}
+		list, err := ioutil.ReadDir(absRoot)
+
+		w := tabwriter.NewWriter(os.Stdout, 12, 1, 3, ' ', 0)
+		fmt.Fprint(w, "ID\tSTARTED\tSTATUS\tPID\tROOTFS\n")
```

Comment:
runc is a bit loose on the optionality of the bundle path.  While runc itself lets it be optional, that's only because it then defaults to "." - but the spec itself says that its a required field.  It could be argued that we should require it on the "start" cmd, if we want to promote spec compliance especially since at the f2f there was a bit of a push to have less defaults being used.

As for rootfs vs bundle in the output.  Irrespective of whether runc saves that info directly or not, it does seem like showing the bundle dir might make more sense since that's the source of all info used to create the container.  Also, from there you can add something like `/config.json` to get that info if needed. Granted from `rootfs` you can remove the tail of it and calculate the bundle dir, but I'm not sure I see the value in forcing people to do that.  The only downside is that to get the `rootfs` dir they'd have to parse the config.json to find its name - but it all depends on which data we think people want to see most often.  We could also just show the rootfs dir name w/o the path as a column.